--- lib/galaxy/eggs/__init__.py	2014-11-16 19:44:25.000000000 -0800
+++ lib/galaxy/eggs/__init__.py	2014-11-16 19:44:43.000000000 -0800
@@ -189,6 +189,7 @@
             else:
                 raise EggNotFetchable( self )
         except pkg_resources.VersionConflict, e:
+            print "EEEEEE", e, "ARGS:", e.args
             if e.args[1].key != e.args[0].key:
                 log.error( "One of Galaxy's managed eggs depends on something which is missing, this is almost certainly a bug in the egg distribution." )
                 log.error( 'Dependency "%s" requires "%s"' % ( e.args[0].project_name, str( e.args[1] ) ) )
